### PR TITLE
[ENG-2801] Updated FOS Message for connected_emails Task

### PR DESCRIPTION
**Pull request by chth0n1x**
- Ticket: [ENG-2801]
- Feature flag: n/a

Purpose:

This ticket looks to provide better feedback displayed to the user by updating the error message when  either 'deactivated' or 'marked as spam' emails would fail to add to a Preprint.

Summary of Changes:

The following file was changed to update the error message: 
-translations/en-us.yml
-app/settings/account/-components/connected-emails/component.ts

Side Effects:

The FOS error message for all 'connected_emails' will display the new, updated string. Any contexts where this message would not be appropriate should create a new or use a different assignment in the 'en-us.yml' file.

Test Plan:

Note: a separate PR is being created for the QUnit test. 

-Login to the staging2 Development OSF.io portal on either Chrome, Firefox, or Edge: "https://staging2.osf.io/profile/".
-Navigate to Preprints and create a Preprint where a user has Admin or Moderator privileges
and at least three others have 'read-only'. These 'read-only' users should have an aggregate of both 'active', 'deactivate', and accounts 'mark(ed) as spam'.
-Optional: Recheck if emails are able to be added by the Admin/Moderator if deactivated accounts are reactivated and accounts marked 'spam' are marked 'non-spam'.
-Optional but recommended: Verify changes in alternative browsers, including Incognito mode.

[ENG-2801]: https://openscience.atlassian.net/browse/ENG-2801

### CODE REVIEW

File: translations/en-us.yml [hunk 1]
```diff
@@ -1812,7 +1812,7 @@ settings:
             save_success: 'Email added'
             title: 'Connected emails'
             unconfirmed_emails: 'Unconfirmed emails'
-            update_fail: 'Unable to update email'
+            update_fail: 'This email has been either deactivated or marked as spam.<br><br>Please email contact@cos.io for further assistance.'
```
Comment (fabmiz) [inline translations/en-us.yml:1815]: Best to use the support email in config `config.support.supportEmail` instead hardcoding it here; also please enclose the email in `a` tags i.e `<a href="mailto:{supportEmail}">{supportEmail}</a>`

**Review by fabmiz — CHANGES_REQUESTED** [review]

File: translations/en-us.yml (hunk 1 above)
Comment (chth0n1x) [reply translations/en-us.yml:1815]: Of course. Let me make those changes, thank you Fabrice.

File: app/settings/account/-components/connected-emails/component.ts [hunk 2]
```diff
@@ -89,7 +91,8 @@ export default class ConnectedEmails extends Component {
 
     @task({ withTestWaiter: true })
     updatePrimaryEmail = task(function *(this: ConnectedEmails, email: UserEmail) {
-        const errorMessage = this.intl.t('settings.account.connected_emails.update_fail');
+        const errorMessage = this.intl.t('settings.account.connected_emails.update_fail',
+            { supportEmail: config.support.supportEmail });
```
Comment (fabmiz) [inline app/settings/account/-components/connected-emails/component.ts:95]: you will need to add `htmlSafe: true` to the options object here to enable rendering the html in the translation.

**Review by fabmiz — CHANGES_REQUESTED** [review]
Looking good. Some minor concerns

**Comment by chth0n1x** [comment]
Added connected-emails/component.ts in the Summary of Changes section.

File: app/settings/account/-components/connected-emails/component.ts [hunk 3]
```diff
@@ -16,6 +16,8 @@ import buildChangeset from 'ember-osf-web/utils/build-changeset';
 import captureException, { getApiErrorMessage } from 'ember-osf-web/utils/capture-exception';
 import getHref from 'ember-osf-web/utils/get-href';
 
+import * as config from 'ember-get-config';
```
[the remark this answers is not in the document]
Comment (chth0n1x) [reply app/settings/account/-components/connected-emails/component.ts:19]: @fabmiz Ok, this gives the linting error we talked about previously. Is there a command to force the commit with error warnings?

File: app/settings/account/-components/connected-emails/component.ts (hunk 2 above)
Comment (chth0n1x) [reply app/settings/account/-components/connected-emails/component.ts:95]: Ok, adding now. Thank you Fabrice.

**Comment by chth0n1x** [comment]
@fabmiz I was about to resolve my merge conflicts but realize I could potentially make a mess of it. Would you be available later to review best practices on this?

File: app/settings/account/-components/connected-emails/component.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (fabmiz) [reply app/settings/account/-components/connected-emails/component.ts]: oh sorry, that was a typo, I meant `import config from 'ember-get-config'` not `import config from 'ember-get'`

**Pull request closed without merging**